src/QueryStrategies/qs_base.jl

```diff
@@ -3,29 +3,63 @@ abstract type QueryStrategy end
 function qs_score end
 
 """
-get_query_object(qs::QueryStrategy, data::Array{T, 2}, pools::Vector{Symbol}, global_indices::Vector{Int}, history::Vector{Int})
+get_query_objects(qs::QueryStrategy, data::Array{T, 2}, pools::Vector{Symbol}, global_indices::Vector{Int}, history::Vector{Int})
 
 # Arguments
 - `query_data`: Subset of the full data set
 - `pools`: Labels for `query_data`
 - `global_indices`: Indices of the observations in `query_data` relative to the full data set.
 - `history`: Indices of previous queries
 """
-function get_query_object end
+function get_query_objects end
 
-function initialize_qs(qs::DataType, model::SVDD.OCClassifier, data::Array{T, 2}, params)::QueryStrategy where T <: Real
-    if qs <: HybridPQs || qs <: HybridQss
-        return qs(model, data; params...)
-    elseif qs <: ModelBasedPQs || qs <: ModelBasedQss
-        return qs(model; params...)
-    elseif qs <: DataBasedPQs || qs <: DataBasedQss
-        kernel = SVDD.get_kernel(model)
-        if typeof(kernel) == MLKernels.GaussianKernel
-            return qs(data, bw_method=MLKernels.getvalue(strategy.kernel.alpha); params...)
+function initialize_qs(qs::DataType, model::SVDD.OCClassifier, data::Array{T, 2}, params::Dict{Symbol, <:Any})::QueryStrategy where T <: Real
+    if qs <: BatchPQs
+        if haskey(params, :SequentialStrategy)
+            sequential_strategy_type = eval(params[:SequentialStrategy][:type])
+            sequential_strategy_params = params[:SequentialStrategy][:param]
+            sequential_strategy = initialize_qs(sequential_strategy_type, model, data, sequential_strategy_params)
+            # use all params except SequentialStrategy for batch query initialization
+            batch_params = filter(tuple -> first(tuple) != :SequentialStrategy, params)
```

Comment:
Move `batch_params` into if statement

Reply:
I don't understand. Both the `if` and the `else` branch use `batch_params`.